modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc

```diff
@@ -0,0 +1,97 @@
+[[contact-methods-pushssh]]
+= SSH Push (with tunnel) contact method
+
+
+SSH Push (with tunnel) ([literal]``ssh-push-tunnel``) is used in environments where clients cannot reach the {productname} Server directly.
+In this environment, clients are located in a firewall-protected zone called a DMZ.
+No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
+
+This SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located on the DMZ.
+After all actions and events are executed, the tunnel is closed.
+
+The server uses SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
+
+
+// FiXME: only here with tunnel, otherwise move to contact-methods-saltssh.adoc
+[IMPORTANT]
+====
+Re-installing systems using the provisioning model is not currently supported on clients managed with SSH Push.
+====
+
+
+[IMPORTANT]
+====
+The tunnel is used to provide the access to the server through the encrypted tunnel.
+The repositories assigned to the SSH Push client (with tunnel) are provided through this tunnel only, thus it is not possible to use package manager tools directly from the client system because the repositories are available only while the tunnel is up.
+In other words, access is only possible if the session is initiated by the server.
+All package managing operations on the client can be performed from the server side only.
+====
+
+For tunneling connections via SSH, a port number is required for tunneling via HTTPS.
+The port number used by default is `1233`.
+To overwrite it, you can add a custom port numbers greater than 1024 to [path]``/etc/rhn/rhn.conf``:
+
+----
+ssh_push_port_https = high_port
+----
+
+
+For security reasons, you might want to use sudo with SSH, to access the system as an unprivileged user instead of as root.
+
+
+.Procedure: Configuring Unprivileged SSH Access
+. Ensure you have the latest [path]``spacewalk-taskomatic`` and [path]``spacewalk-certs-tools`` packages installed on the {productname} Server.
```

Comment:
Is it really possible to have the server with no such packages installed?

Reply:
No idea, probably not.  Maybe, "latest" is the important word.  I do not know how important it is here to tell the user that the latest MU with all security related updates should be installed.

Reply:
Even with focus on word "latest" it sounds bit strange. Maybe this sentance appeared when the change with such feature appears, I would suggest to drop it.

Reply:
Ok, makes sense, IMO.  Done.